packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -109,8 +109,14 @@ class Plugin implements CLNPlugin {
   /// configuration.
   Future<Map<String, Object>> init(Plugin plugin, Map<String, Object> request) {
     // TODO: store the configuration inside the plugin (it is inside the request)
+    var response = HashMap<String, Object>();
+    Map<String, Object> pluginOption = request['options'] as Map<String, Object>;
+    Map<String, Object> configuration = request['configuration'] as Map<String, Object>;
+    response['options'] = pluginOption;
+    response['configuration'] = configuration;
+    
     // TODO: get the option value inside the request and assign it to the options in some way!
-    return Future.value({});
+    return Future.value(response);
```

Comment:
Note that the configuration is to configure the plugin and not to return to core lightning.

From docs
>The plugin must respond to init calls. The response should be a valid JSON-RPC response to the init, but this is not currently enforced. If the response is an object containing result which contains disable then the plugin will be disabled and the contents of this member is the reason why.

In other words the pluginOption and the configuration are propeities of the class!